Playdown.swift

```diff
@@ -43,85 +40,84 @@ class StreamReader  {
     
     /// Return next line, or nil on EOF.
     func nextLine() -> String? {
-        precondition(fileHandle != nil, "Attempt to read from closed file")
```

Comment:
Oops - I meant actually to remove that because fileHandle is no longer optional and that gives an error. Would you prefer I keep it as an optional value?